.ci_support/linux_aarch64_.yaml

```diff
@@ -0,0 +1,8 @@
+c_compiler:
```

Comment:
We had to add a few things here

```
cdt_arch: aarch64
cdt_name: cos7
BUILD: aarch64-conda_cos7-linux-gnu
target_platform: linux-aarch64
```
not sure if `target_platform` is necessary, but the others are because the compilers don't target `cos6` so you have to change what conda-build thinks is correct.

Reply:
I notice in my notes, that I didn't have to do this for zlib, but it will become important for some other packages.
https://github.com/Archiconda/zlib-feedstock/blob/aarch64/.ci_support/linux_.yaml

Reply:
can you make a pr to conda-forge-pinning ?

Reply:
A small patches to conda-build should make these unnecessary.  I'll submit a PR today with the changes.

Reply:
conda/conda-build#3357 sets these as the default for the linux-aarch64 platform.